jxls/src/main/java/org/jxls/builder/xls/XlsCommentAreaBuilder.java

```diff
@@ -219,11 +224,104 @@ public static void addCommandMapping(String commandName, Class<? extends Command
         return userAreas;
     }
 
+    private void extractLiterals(CellData cellData, String text, List<String> literalList) {
+		String literal = "";
+		for (int i = 0, n = text.length(); i < n; i++) {
+
+			// if the first 3 chars of literal are 'jx:'
+			if (literal.length() >= 3
+					&& literal.substring(literal.length() - 3, literal.length()).equals(COMMAND_PREFIX)) {
+
+				// save comment parsed before
+				literal = literal.substring(0, literal.length() - 3);
+				makeCommentLines(literal, literalList);
+				String remainingText = text.substring(i - 3, n);
+				String cmd = makeOperators(cellData, remainingText);
+				literal = "";
+				literalList.add(cmd);
+				i = i + cmd.length() - 3;
+
+				// if text no ending
+				if (i != n)
+					literal = String.valueOf(text.charAt(i));
+				else
+					literal = "";
+
+			} else {
+				// else is the next char to new literal
+				literal = literal + String.valueOf(text.charAt(i));
+			}
+		}
+
+		// last comment
+		if (!literal.isEmpty()) {
+			makeCommentLines(literal, literalList);
+		}
+	}
+    
+	private void makeCommentLines(String literal, List<String> literalList) {
+		literalList.addAll(
+				Arrays.stream(literal.split("\n"))
+					.filter(line -> ! line.isEmpty())
+					.collect(toList()));
+	}
+	
+	private String makeOperators(CellData cellData, String text) {
+		// result command
+		String cmd = "";
+		Stack<Character> bracketsStack = new Stack<>();
+		Stack<Character> quotationsStack = new Stack<>();
+
+		for (int i = 0; i < text.length(); i++) {
+			char symbol = text.charAt(i);
+
+			if (symbol == '"' || symbol == '\'')
+				if (!quotationsStack.empty() && quotationsStack.peek() == symbol)
+					quotationsStack.pop();
+				else
+					quotationsStack.push(symbol);
+
+			switch (symbol) {
+			case '(':
+				if (quotationsStack.empty())
+					bracketsStack.push(symbol);
+				break;
+			case ')':
+				if (quotationsStack.empty()) {
+					if (bracketsStack.peek() == '(') {
+						bracketsStack.pop();
+						if (bracketsStack.empty())
+							return cmd + symbol;
+					} else {
+						// no opening, but closing !!
+						String errorMessage = "Got closing ) but no opening of it. Pos: " + text.substring(0, i);
+						logger.error(errorMessage);
```

Comment:
I think there's no need to log an error if you also throw an exception. It might be the responsibility of the catcher to log the error.